submissions/ArthurGorbenko/Friend's app/js/script.js

```diff
@@ -0,0 +1,282 @@
+const CONSTS = {
+  LIST_OF_CARDS: document.querySelector(".list"),
+  DIRECT_MESSAGE_FORM: document.querySelector(".direct-message"),
+  FORM: document.querySelector(".form"),
+  SEND_BUTTON: document.querySelector(".send-on-click-button"),
+  SEARCH_FIELD: document.querySelector(".search-field"),
+  RADIO_AGE_FORM: document.querySelector(".wrapper-radio-age"),
+  RADIO_GENDER_FORM: document.querySelector(".wrapper-radio-gender"),
+  EMAIL_FIELD: document.querySelector(".email-field"),
+  TEXT_AREA: document.querySelector(".textarea"),
+  ALERT_MESSAGE: document.querySelector(".alert-message"),
+  RESET_BUTTON: document.querySelector(".reset"),
+  HAMBURGER: document.querySelector(".hamburger.hamburger--slider"),
+  THEME_WRAPPER: document.querySelector(".wrapper-theme-change"),
+  BODY: document.querySelector(".default"),
+  MENU: document.querySelector(".menu"),
+  WRAPPER_MENU: document.querySelector(".wrapper-menu")
+};
+
+const MOBILE_RESOLUTION = "(max-width: 599px)";
+const SORT_BY_ASCENDING = "ascending";
+const CARD_CLASS = "card";
+const BUTTON_CLASS = "button";
+const DARK_INPUT_ID = "dark";
+
+let filteredByGender = false;
+let radioButtons = document.querySelectorAll(".visually-hidden");
+let usersConst;
+let users;
+
+const letterToUpperCase = word => {
+  return word.charAt(0).toUpperCase() + word.slice(1);
+};
+
+const refreshDMWindow = () => {
+  CONSTS.FORM.classList.remove("visually-hidden");
+  CONSTS.ALERT_MESSAGE.classList.add("visually-hidden");
+  CONSTS.TEXT_AREA.value = "";
+};
+
+let menuToggled = false;
+
+const moveMenu = () => {
```

Comment:
I checked this on your demo, please check screen http://prntscr.com/ocgusq. 
i will recommend you make this moving by css. It's really easy and doesn't need extra logic. From js you only add classes for opening menu. Other stuff like moving, hiding and sizes with media query you can make with CSS.
Check this. 
https://codepen.io/shieldsma91/pen/zLpbLX
No need to make on pure CSS, but use styles where it can. It will be match more better.

Reply:
Check please my approach, is it ok?I left eventListener but all animations work on CSS.